pkg/image/api/types.go

```diff
@@ -60,6 +77,30 @@ type ImageLayer struct {
 	Size int64
 }
 
+// SignatureValidationResult holds a result of signature validation.
+type SignatureValidationResult string
+
+// ImageSignature holds a signature of an image.
+type ImageSignature struct {
```

Comment:
> Of course if the public key is trusted per policy, the policy will have a copy of the key. Or we could ask a key server, and perhaps get a copy.

I think the field is useful enough to have and if the server has sufficient information to complete it, it should do so.  Knowing it was signed by `openshift-builder` or `mycompany-operations` is valuable.